README.md

```diff
@@ -1,39 +1,80 @@
-# collection_template
-You can build a new repository for an Ansible Collection using this template by following [Creating a repository from a template](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template). This README.md contains recommended headings for your collection README.md, with comments describing what each section should contain. Once you have created your collection repository, delete this paragraph and the title above it from your README.md.
 
-# Foo Collection
-<!-- Add CI and code coverage badges here. Samples included below. -->
-[![CI](https://github.com/ansible-collections/REPONAMEHERE/workflows/CI/badge.svg?event=push)](https://github.com/ansible-collections/REPONAMEHERE/actions) [![Codecov](https://img.shields.io/codecov/c/github/ansible-collections/REPONAMEHERE)](https://codecov.io/gh/ansible-collections/REPONAMEHERE)
 
-<!-- Describe the collection and why a user would want to use it. What does the collection do? -->
+# Community Network Collection
+[![Run Status](https://api.shippable.com/projects/5e7c598d1c7dc1000762800c/badge?branch=master)](https://app.shippable.com/github/ansible-collections/community.network/dashboard) <!--[![Codecov](https://img.shields.io/codecov/c/github/ansible-collections/vyos)](https://codecov.io/gh/ansible-collections/community.network)-->
```

Comment:
```suggestion
[![Run Status](https://api.shippable.com/projects/5e7c598d1c7dc1000762800c/badge?branch=master)](https://app.shippable.com/github/ansible-collections/community.network/dashboard) [![Codecov](https://img.shields.io/codecov/c/github/ansible-collections/community.network)](https://codecov.io/gh/ansible-collections/community.network)
```

Reply:
community.network isn't up on codecov yet so keeping this commented out until it is. I'll correct the vyos thing tho :-)

Reply:
It should be up tomorrow. I found out today that it used a wrong token.

Reply:
It did work, it is no longer empty :)